account_invoice_margin/models/account_invoice.py

```diff
@@ -40,7 +44,7 @@ def _compute_margin(self):
             margin = 0.0
             margin_signed = 0.0
             price_subtotal = 0.0
-            for line in invoice.invoice_line_ids:
+            for line in invoice._get_margin_applicable_lines():
                 if any(line.sale_line_ids.mapped("is_downpayment")):
```

Comment:
You should remove this and if you can do the other change as well in the same PR.

Reply:
Yes. I am on it.